deploy/helm/sumologic/Chart.yaml

```diff
@@ -33,7 +33,7 @@ dependencies:
     repository: https://sumologic.github.io/tailing-sidecar
     condition: tailing-sidecar-operator.enabled
   - name: opentelemetry-operator
-    version: 0.56.1
+    version: 0.76.0
```

Comment:
Installs operator version `v0.115.0`

Reply:
Probably insignificant, but [the relevant release page](https://github.com/open-telemetry/opentelemetry-helm-charts/releases/tag/opentelemetry-operator-0.76.0) says it's operator `0.115.0`.

Reply:
When I deploy this subchart it installs `0.115.1` on my machine, they must have released a patch.